test/library/standard/Path/victor-ludorum/testcases_isAbsPath.chpl

```diff
@@ -13,3 +14,19 @@
    writeln(isAbsPath("\\server"));
 	 	 	 
    writeln(isAbsPath("."));
+
+   // Check to see if we correctly detect Pythons's abspath to the $CWD as an
+   // absolute path (This is to help identify platforms we don't support.)
+   writeln(isAbsPath(getPythonsAbspath()));
+
+   proc getPythonsAbspath() {
+     var command = "python -c 'import os; print(os.path.abspath(\"\"))'";
```

Comment:
Couldn't you just use [getcwd](https://docs.python.org/2/library/os.html#os.getcwd)?